package.json

```diff
@@ -4,8 +4,8 @@
   "name": "@autorest/csharp-v3",
   "description": "See readme.md for instructions",
   "scripts": {
-    "start": "dotnet ./bin/AutoRest.CSharp.V3.dll --server",
-    "debug": "dotnet ./bin/AutoRest.CSharp.V3.dll --server --launch-debugger"
+    "start": "dotnet ./artifacts/bin/AutoRest.CSharp.V3/Debug/netcoreapp3.0/AutoRest.CSharp.V3.dll  --server",
+    "debug": "dotnet ./artifacts/bin/AutoRest.CSharp.V3/Debug/netcoreapp3.0/AutoRest.CSharp.V3.dll --server --launch-debugger"
```

Comment:
This locks us into always running the debug version of the package. I intentionally made everything build into `bin` flat because:
- When shipping this extension, you can build it as Release and it will still work.
- If we update to netcoreappX.X, this also wouldn't need to change. The framework is basically irrelevant for the extension.
- AFAIK, there aren't multiple `start`s. So, we can't have a `start-debug` or `start-release` or something like that. Otherwise, you'd have to ship a different package.json than you are using for development.

Other than it being the standard convention to output to `[buildName]/[framework]`, let me know if there are other reasons for this. I just wanted to keep it simple for now.

Reply:
This `package.json` is not in a publishable state anyway and we would have to fix that in the future. For now I want to bring the repo structure to what we have in other repos.

Reply:
That's fine. When we do get to publishing, it would likely make sense to keep it flatter as we likely won't publish multiple DLLs of different frameworks into the same package. But, we're not at the point of worrying about that anyway.

Reply:
Yes, but your concerns seem to be about things that would only matter when publishing.

Reply:
I'm saying it is fine. I don't mind if we nest it like that for now. I was just pointing out why I didn't nest it originally, and was wondering what benefit there was to nesting it.